src/data/BackendFactory.hpp

```diff
@@ -51,8 +51,7 @@ make_Backend(util::Config const& config)
     auto const type = config.value<std::string>("database.type");
     std::shared_ptr<BackendInterface> backend = nullptr;
 
-    // TODO: retire `cassandra-new` by next release after 2.0
-    if (boost::iequals(type, "cassandra") or boost::iequals(type, "cassandra-new")) {
+    if (boost::iequals(type, "cassandra")) {
```

Comment:
This is a breaking change for config. Are you sure it is ok to do that?

Reply:
👍 Good call. 
I think "cassandra-new" is only used internally, when we compared the performance between scylladb and cassandra. Will check.

Reply:
Checked with @godexsoft . It is never announced, should be safe to delete.